api/config/config_staging.json

```diff
@@ -58,7 +58,8 @@
     "enableComplianceTraining": true,
     "enableEraCommons": true,
     "enableDataUseAgreement": false,
-    "enableBetaAccess": true
+    "enableBetaAccess": true,
+    "unsafeAllowSelfBypass": true
```

Comment:
Works for me. Will change. There won't be a button for it? I think it should probably just be script accessible.

Reply:
I'm not sure I agree (though I don't feel too strongly, since the immediate use cases are only in test).

My logic goes: this feature is meant for testing, and staging is where we're doing QA testing, so it should probably be enabled there. It would make certain flows easier by allowing new users to quickly bypass the access modules and gain access to the workbench. (It's not a huge issue that Rainforest users might see the self-bypass UI; they are required to follow the script, so they will either use it or not depending on how the test is described.)

See more notes in https://precisionmedicineinitiative.atlassian.net/browse/RW-2679

Reply:
The ticket says "one-click", I assume this means there are plans for a UI-driven flow to run this yourself - if not, I'm not convinced this PR makes sense. In a script-based approach it would be just as easy to authenticate as an admin user (and/or grant the user admin privileges).

I see your point, but unless there's a concrete use case for this for Rainforest in staging, I'd favor having them QA an environment that's closer configured to what we're doing in prod for now - this also matches prior behavior of `enforceRegistered`.

Reply:
Feeling this through, a UI-driven flow for self-bypass makes sense to me. (So you don't need to go back to the command line and run the command, possibly interrupting flow.)

I am checking if it is a requirement for the MVP for Perf testing, or if it can be a follow on task for us.

Reply:
We're going to follow up with a click to self bypass, and enforcing registered in test.